README.md

```diff
@@ -95,6 +97,30 @@ If you do not have a working Golang environment, please follow [How to install G
 go get -u github.com/minio/mc
 ```
 
+## Configure
+
+By default `mc` is configured to interact with the _play_ environment, a public Minio server. If you want to configure a different server:
+
+```sh
+mc config host add <ALIAS> <YOUR-S3-ENDPOINT> <YOUR-ACCESS-KEY> <YOUR-SECRET-KEY>
+```
+
+You will be able to list buckets (i.e. `mc ls <ALIAS>`) or any other available operation.
+
```

Comment:
BTW this looks like a repeated mention.. i.e there are details about adding S3 and GCS below, we don't need to repeat the `#Configure` section. 

Perhaps you can move the `#Docker` section after `# Test your setup` section underneath.